packages/tour-kit/src/types.ts

```diff
@@ -84,6 +84,19 @@ export interface Config {
 	closeHandler: CloseHandler;
 	isMinimized?: boolean;
 	options?: Options;
+	placement?:
```

Comment:
@moon0326 I think we can use the `Placement` type from `@popperjs/core`. 

```ts
import * as PopperJS from '@popperjs/core';


	placement?: PopperJS.Placement;
```

Reply:
Good suggestion! 👍 updated in https://github.com/Automattic/wp-calypso/pull/64317/commits/03cfe66350873cee27c4a4e775036a5f58268483